services/__tests__/tokens.test.js

```diff
@@ -73,6 +73,21 @@ describe('createToken', () => {
     });
   });
 
+  test('with allowedApplications', () => {
+    tokens.createToken({
+      allowedApplications: [{ platform: 'iOS', bundleId: 'com.example.foo' }]
+    });
+    expect(tu.requestConfig(tokens)).toEqual({
+      path: '/tokens/v2/:ownerId',
+      method: 'POST',
+      params: {},
+      body: {
+        allowedApplications: [{ platform: 'iOS', bundleId: 'com.example.foo' }],
+        scopes: []
```

Comment:
I think it's fine if we drop `scopes` from this request body (?)

Reply:
Empty scopes array is included in the request body even if its omitted from the SDK call itself. Thus it needs to be there for the assertion to pass.